mars/services/task/execution/ray/context.py

```diff
@@ -144,6 +153,11 @@ def get_chunks_meta(
             result.append(meta)
         return result
 
+    @implements(Context.get_total_n_cpu)
+    def get_total_n_cpu(self) -> int:
+        # TODO(fyrestone): Support auto scaling.
+        return self._config.get_n_cpu() * self._config.get_n_worker()
```

Comment:
- What happed if `n_cpu` and `n_worker` not configured into ray config?
- Seems no case cover this

Reply:
Why `mars` doesn't need to implement `get_total_n_cpu`?

Reply:
> * What happed if `n_cpu` and `n_worker` not configured into ray config?
> * Seems no case cover this

The `n_cpu` and `n_worker` will be set when starting local cluster: https://github.com/mars-project/mars/blob/master/mars/deploy/oscar/local.py#L162. If these fields are not set, there should be a bug.

Mars backend has implemented the `get_total_n_cpu` API: https://github.com/mars-project/mars/blob/master/mars/services/context.py#L113